packages/yarnpkg-core/sources/scriptUtils.ts

```diff
@@ -30,11 +29,9 @@ enum PackageManager {
 
 async function makePathWrapper(location: PortablePath, name: Filename, argv0: NativePath, args: Array<string> = []) {
   if (process.platform === `win32`) {
-    await Promise.all([
-      xfs.writeFilePromise(ppath.format({dir: location, name, ext: `.exe`}), getBinjumper()),
-      xfs.writeFilePromise(ppath.format({dir: location, name, ext: `.exe.info`}), [argv0, ...args].join(`\n`)),
-      xfs.writeFilePromise(ppath.format({dir: location, name, ext: `.cmd`}), `@"${argv0}" ${args.map(arg => `"${arg.replace(`"`, `""`)}"`).join(` `)} %*\n`),
-    ]);
+    // https://github.com/microsoft/terminal/issues/217#issuecomment-737594785
+    const cmdScript = `@echo off\ngoto #_undefined_# 2>NUL || title %COMSPEC% & "${argv0}" ${args.map(arg => `"${arg.replace(`"`, `""`)}"`).join(` `)} %*`;
```

Comment:
`@echo off` changes the caller's echo status. Using leading `@` for the commands is more "hygenic".

```suggestion
    const cmdScript = `@goto #_undefined_# 2>NUL || @title %COMSPEC% & @"${argv0}" ${args.map(arg => `"${arg.replace(`"`, `""`)}"`).join(` `)} %*`;
```